### DISCUSSION TITLE
为什么不能运行

### DISCUSSION THREAD

**Discussion by chenlianglian** [eggjs/egg#4921] [Q&A]
拉取下来后安装完依赖执行npm run site:dev等命令后，终端响应的是'APP_ROOT' 不是内部或外部命令，也不是可运行的程序
或批处理文件
请问大家都是怎么解决的
<img width="297" alt="1" src="https://user-images.githubusercontent.com/27852531/162378645-fdbc4b46-ed56-4347-8212-b69a743a40e3.png">
？

**Comment by killagu** [comment]
Please use [cross-env](https://www.npmjs.com/package/cross-env).

```json
{
  "scripts": {
    "site:dev": "cross-env APP_ROOT=./site dumi dev"
  }
}
```
